Remotion/Validation/Core/Validators/LengthValidator.cs

```diff
@@ -15,31 +26,34 @@ public class LengthValidator : PropertyValidator, ILengthValidator
 
     public int Max { get; }
```

Comment:
Hmm...I think we should use int? for Max. later on, this might get refactored to using three separate types.